common/traits/hfe_species_trait.txt

```diff
@@ -33,78 +34,128 @@ hfe_species_trait_war_fans = {
 	leader_age_min = 10
 	icon = gfx/interface/icons/other/hfe_war_fans.dds
 	allowed_archetypes = {
-		BIOLOGICAL
+		BIOLOGICAL LITHOID
 	}
 	ai_weight = {
 		weight = 100
 	}
 	modifier = {
 		admiral_skill_levels = 1
 		general_skill_levels = 1
-		hfe_planet_jobs_war_produces_mult = 0.3
+		pop_ethic_authoritarian_attraction_mult = 1
 		pop_ethic_militarist_attraction_mult = 0.5
+		pop_ethic_xenophile_attraction_mult = 0.3
+		army_morale = 0.25
 	}
 }
 
 hfe_species_trait_error = {
-	cost = 1
+	cost = -1
 	initial = no
 	randomized = no
 	modification = no
 	sorting_priority = 25
 	leader_age_min = 10
 	icon = gfx/interface/icons/other/hfe_error.dds
+	potential_crossbreeding_chance = 1
+	slave_cost = {
+		energy = -250
+	}
 	allowed_archetypes = {
-		BIOLOGICAL
+		BIOLOGICAL LITHOID
 	}
 	ai_weight = {
-		weight = 100
+		weight = 0 # This is a bad Trait!
 	}
 	modifier = {
 		pop_growth_speed = 0.5
 		planet_jobs_engineering_research_produces_mult = -0.5
 		planet_jobs_physics_research_produces_mult = -0.5
 		planet_jobs_society_research_produces_mult = -0.5
 		planet_jobs_energy_produces_mult = -0.5
+		pop_happiness = 0.1 # Happy to be alive
+		pop_amenities_usage_add = 1 # More medicine
+		pop_cat_bio_trophy_happiness = 0.5
+		leader_age = -20
 	}
 }
 
 hfe_species_trait_happy_slave = {
-	cost = 1
+	cost = 0
 	initial = no
 	randomized = no
-	modification = no
+	modification = yes # Only for Ju Ahets
```

Comment:
No, this is lost knowledge

Reply:
They can only be used by the _awakened_ Ju Ahets. If you want it to be _permanently_ lost, I will change it.